### PR TITLE
calendar created 1

### PR DESCRIPTION
**Pull request by pr1syazhniy**
[DEMO LINK](https://pr1syazhniy.github.io/layout_calendar/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="//fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--1">
```
Comment (vlad-demchuk) [inline src/index.html]: month-length--1?

Comment (vlad-demchuk) [reply src/index.html]: it should be 28-31

File: src/styles/utils/normalize.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+div {
+  list-style: none;
```
Comment (vlad-demchuk) [inline src/styles/utils/normalize.scss]: `div` doesn't have `list-style`

specify elements you want to reset

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
add a demo link to pr description

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
check your deploy, please
demo link still not works

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Screenshot 2023-04-04 at 18 18 06
Almost good, but seems that your week has only 6 days (6 squares per row) 😉

**Review by Viktor-Kost — APPROVED** [review]
nice!

**Pull request closed without merging**